Sprint-3/implement/get-card-value.js

```diff
@@ -29,3 +29,30 @@
 // Given a card with an invalid rank (neither a number nor a recognized face card),
 // When the function is called with such a card,
 // Then it should throw an error indicating "Invalid card rank."
+function getCardValue(card){
+    let rank = card.slice(0,-1)
+    if(rank === "A"){
+        return 11
+    }
+     const numericValue = parseInt(rank)
+    if(numericValue >= 2 && numericValue < 10){
+        return  numericValue
+    }
+    const faceCards = ["10", "J", "Q", "K"]
+     if(faceCards.includes(rank)){
+        return 10
+    }
+    return "Invalid card rank."
+
+}
+console.log(getCardValue("6♠"))
+
+const currentOutPut = getCardValue("A♠")
+const targetOutPut = 11
+console.assert(currentOutPut === targetOutPut,`currenOutPut:${currentOutPut},targetOutPut:${targetOutPut}`) 
+
+const currentOutPut2 = getCardValue("5♠")
```

Comment:
in getCardValue("0Q♠");
getCardValue("010♠");
getCardValue("02♠"); it works, but instead of throwing an error, it returns an error, so I updated that one.
getCardValue("2.1♠") gives unexpected results . i updated it as well